app/domain/authentication/authn_jwt/id_from_decoded_token_provider.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+    # Class for providing jwt identity from the decoded token from the field specified in a secret
+    class IdentityFromDecodedTokenProvider < IdentityProviderInterface
+
+      attr_accessor :secret_fetcher
```

Comment:
Why do we expose this ?

Reply:
I wanted to expose it for tests to mock this. But its not needed